frontend/public/locales/en/translation.json

```diff
@@ -27,6 +27,8 @@
   "{{remaining}} more": "{{remaining}} more",
   "{{remoteCount}} Remote": "{{remoteCount}} Remote",
   "{{remoteCount}} Remote, 1 Local": "{{remoteCount}} Remote, 1 Local",
+  "{{testInventory}} is not an existing ansible inventory": "{{testInventory}} is not an existing ansible inventory",
+  "{{testSecret}} is not an existing ansible credential": "{{testSecret}} is not an existing ansible credential",
```

Comment:
1. Ansible should always start with capital A.
2. Why are interpolation variables called `testX`? `{{inventory}}` and `{{credential}}` would be the natural choices.
3. Why is the name of the credential having the first letter uppercased? (See screenshot.)

<img width="440" alt="image" src="https://user-images.githubusercontent.com/42188127/232568580-e6a93263-46aa-402e-a2b8-bbc50cce54a4.png">

Reply:
done, done, done